Packages/com.chisel.components/Chisel/Components/API.public/BrushMeshAssetFactory/LinearStairs/BrushFactory.LinearStairs.cs

```diff
@@ -395,23 +399,23 @@ public static bool GenerateLinearStairsSubMeshes(CSGBrushSubMesh[] subMeshes, CS
 					}
 					//if (!haveLeftSideDown)
 					{
-						var height		= definition.stepHeight;
+						var stepHeight		= definition.stepHeight;
```

Comment:
stepHeight was a more accurate description. I kept finding this "height" when looking for the height of the stairs in the file.